code/datums/wounds/robotic/robotic_blunt.dm

```diff
@@ -0,0 +1,1027 @@
+/datum/wound/blunt/robotic/critical/proc/plunge(obj/item/plunger/treating_plunger, mob/user)
+	if (!treating_plunger.tool_use_check())
+		return TRUE
+
+	var/their_or_other = (user == victim ? "their" : "[user]'s")
+	if (user)
+		user.visible_message(span_notice("[user] starts plunging at the dents on [their_or_other] [limb.plaintext_zone]..."))
+
+	var/delay_mult = 1
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		delay_mult *= 0.75
+
+	delay_mult /= treating_plunger.plunge_mod
+
+	if (!treating_plunger.use_tool(target = victim, user = user, delay = 8 SECONDS * delay_mult, volume = 50, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
+		return TRUE
+
+	var/chance = 80
+	if (victim == user)
+		chance *= 0.6
+
+	if (HAS_TRAIT(user, TRAIT_KNOW_ROBO_WIRES))
+		chance *= 1.25
+	if (HAS_TRAIT(user, TRAIT_KNOW_ENGI_WIRES))
+		chance *= 1.1
+	if (HAS_TRAIT(user, TRAIT_DIAGNOSTIC_HUD))
+		chance *= 1.25
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		chance *= 1.5
+
+	if (prob(chance))
+		if (user)
+			user.visible_message(span_green("[victim]'s [limb.plaintext_zone] lets out a sharp POP as the suction of [treating_plunger] forces the superstructure into it's normal position!"))
+		to_chat(victim, span_green("Your [limb.plaintext_zone]'s structure has been reset to it's proper position! Your next step is to secure it with a screwdriver/wrench, though bone gel would also work."))
+		set_superstructure_status(TRUE)
+	else
+		if (user)
+			user.visible_message(span_warning("[victim]'s [limb.plaintext_zone] lets out a strained creak as [treating_plunger] rips some shrapnel out of the chassis!"))
+		limb.receive_damage(brute = 5, damage_source = treating_plunger)
+	return TRUE
+
+/datum/wound/blunt/robotic/critical/handle_percussive_maintenance_success(attacking_item)
+	victim.visible_message(span_green("[victim]'s [limb.plaintext_zone] gets smashed into a proper shape!"), \
+		span_green("Your [limb.plaintext_zone] smashes into place! Your next step is to secure it with a screwdriver/wrench, though bone gel would also work."))
+	set_superstructure_status(TRUE)
+
+/datum/wound/blunt/robotic/critical/handle_percussive_maintenance_failure(attacking_item)
+	to_chat(victim, span_warning("Your [limb.plaintext_zone] only deforms more from the impact..."))
+	limb.receive_damage(brute = 1, damage_source = attacking_item, wound_bonus = CANT_WOUND)
+
+/datum/wound/blunt/robotic/critical/uses_percussive_maintenance()
+	return (!superstructure_remedied && limb_malleable())
+
+/datum/wound/blunt/robotic/critical/proc/set_superstructure_status(remedied)
+	superstructure_remedied = remedied
+	ready_to_secure_internals = remedied
+
+#undef BLUNT_ATTACK_DAZE_MULT
+
+#undef PERCUSSIVE_MAINTENANCE_REPAIR_CHANCE
+#undef PERCUSSIVE_MAINTENANCE_DAMAGE_THRESHOLD
+
+#undef ROBOTIC_T3_BLUNT_WOUND_RCD_COST
+
+#undef ROBOTIC_WOUND_DETERMINATION_MOVEMENT_EFFECT_MOD
+#undef ROBOTIC_WOUND_DETERMINATION_HIT_DAZE_MULT
+#undef ROBOTIC_WOUND_DETERMINATION_HIT_NAUSEA_MULT
```

Comment:
Wow good golly this file is 1027 lines of code long, almost exactly double the longest existing wound file, `bones.dm`. Really makes you consider how much complexity these specific wounds have for how relatively rare they will probably be compared to normal bone wounds.

Reply:
Yes, its more complicated, but its A. a blunt wound, thats part of its archtype, B. A lot of the complexity is in the BACKEND. It's significantly less complex than it actually seems, and all the little bits and bobs are documented in-game, _unlike existing wounds_, which often dont even mention things like being able to grasp a limb, or rest to reduce bleedrate - hell, I dont even they bleed wounds mention the fact epipens slow their bleeding.